framework/branch-key-store.md

```diff
@@ -844,7 +820,7 @@ the keystore operation MUST call with a request constructed as follows:
   If the KMS Configuration is MRDiscovery, `KeyId` MUST be the `kms-arn` attribute value of the AWS DDB response item, with the region replaced by the configured region.
   Otherwise, it MUST BE the Keystore's configured KMS Key.
 - `CiphertextBlob` MUST be the `enc` attribute value on the AWS DDB response item
-- `EncryptionContext` MUST be the [encryption context](#encryption-context) constructed above
+- `EncryptionContext` MUST be the [AWS KMS Encryption Context](#aws-kms-encryption-context) constructed above
```

Comment:
I don't think you construct that above? But you have the branch key context, and you can use #encryption-context-from-authenticated-branch-key-context